src/Features/CSharp/Portable/Completion/KeywordRecommenders/IntoKeywordRecommender.cs

```diff
@@ -40,6 +40,17 @@ private bool IsValidContextForSelect(CSharpSyntaxContext context)
 
             var lastToken = select.Expression.GetLastToken(includeSkipped: true);
 
+            var memberAccessExpression = select.Expression as MemberAccessExpressionSyntax;
+            if(memberAccessExpression  != null && string.IsNullOrEmpty(memberAccessExpression.Name.ToString()))
+            {
+                return false;
+            }
+
+            //if(lastToken.Kind() == SyntaxKind.DotToken)
```

Comment:
If you look at other recommenders they follow a pattern to get the first 'complete' token before the caret.  You want this so that you don't offer "into" when you have ```.``` or ```.i```